Makefile.defs

```diff
@@ -5,9 +5,10 @@ CONFDIR?=/etc
 
 INSTALL = install
 
+NATIVE_BYTE_ORDER=$(shell lscpu | grep Endian |awk '{ print $$3"_"$$4 }')
```

Comment:
`lscpu` may not be available everywhere, what about we use something like this instead?
```
const INT_SIZE int = int(unsafe.Sizeof(0))
var i int = 0x1
bs := (*[INT_SIZE]byte)(unsafe.Pointer(&i))
if bs[0] == 0 {
    // big endian
} else {
    // little endian
}
```